libcodechecker/server/client_db_access_handler.py

```diff
@@ -359,14 +359,13 @@ def __init__(self,
         self.__Session = Session
         self.__storage_session = StorageSession()
 
-    def __lastBugEventPos(self, report_id):
+    def __lastBugEventPos(self, session, report_id):
```

Comment:
We don't use this function anywhere. Do we want to keep it?

Reply:
I've removed it.